<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="de_DE">
<context>
    <name>UserError</name>
    <message>
        <location filename="../modules/general.py" line="354"/>
        <source>Multiple layers selected.</source>
        <translation>Mehrere Layer ausgewählt. Bitte nur einen Layer für die Verarbeitung wählen.</translation>
    </message>
    <message>
        <location filename="../modules/general.py" line="358"/>
        <source>No layer selected.</source>
        <translation>Kein Layer ausgewählt. Bitte einen Layer für die Verarbeitung wählen.</translation>
    </message>
    <message>
        <location filename="../modules/general.py" line="364"/>
        <source>Selected node is not a layer.</source>
        <translation>Das ausgewählte Objekt ist kein Layer. Es können nur Vektorlayer mit Linien-Geometrie verarbeitet werden.</translation>
    </message>
    <message>
        <location filename="../modules/general.py" line="370"/>
        <source>Selected layer is not a vector layer.</source>
        <translation>Der gewählte Layer ist kein Vektorlayer. Es können nur Vektorlayer mit Linien-Geometrie verarbeitet werden.</translation>
    </message>
    <message>
        <location filename="../modules/general.py" line="375"/>
        <source>The selected layer is not a line layer.</source>
        <translation>Der gewählte Layer ist kein Linien-Layer. Es können nur Vektorlayer mit Linien-Geometrie verarbeitet werden.</translation>
    </message>
    <message>
        <location filename="../modules/general.py" line="58"/>
        <source>No QGIS project is currently open.</source>
        <translation>Kein QGSI-Projekt geöffnet.</translation>
    </message>
    <message>
        <location filename="../modules/general.py" line="101"/>
        <source>Project is not saved. Please save the project first.</source>
        <translation>Das Projekt ist noch nicht gespeichert. Bitte das Projekt abspeichern, damit ein Projektordner erstellt wird.</translation>
    </message>
</context>
<context>
    <name>feature_note</name>
    <message>
        <location filename="../modules/t_intersection_analyzer.py" line="42"/>
        <source>Could not determine main pipe.</source>
        <translation>Konnte Hauptleitung nicht definieren.</translation>
    </message>
    <message>
        <location filename="../modules/t_intersection_analyzer.py" line="56"/>
        <source>Could not determine main pipe endpoints.</source>
        <translation>Konnte die Endpunkte der Hauptleitung nicht finden.</translation>
    </message>
    <message>
        <location filename="../modules/t_intersection_analyzer.py" line="78"/>
        <source>Main pipe: {0}</source>
        <translation>Hauptleitung: {0}</translation>
    </message>
    <message>
        <location filename="../modules/poi_classifier.py" line="75"/>
        <source>Intersection without endpoints - lines must be devided.</source>
        <translation>Schnittpunkt ohne Verbindung - die Linien müssen mit Endpunkten verbunden werden.</translation>
    </message>
    <message>
        <location filename="../modules/poi_classifier.py" line="147"/>
        <source>More than three lines at intersection - split up intersection in multiple points.</source>
        <translation>Mehr als 3 Lininen an Knoten - bitte Knoten auflösen.</translation>
    </message>
    <message>
        <location filename="../modules/poi_classifier.py" line="196"/>
        <source>Unconnected line - make sure the line is connected to the network.</source>
        <translation>Linie nicht an Netz angeschlossen - bitte Verbinddungsstelle prüfen.</translation>
    </message>
    <message>
        <location filename="../modules/t_intersection_analyzer.py" line="322"/>
        <source>Non-standard pipe dimension detected for reducer.</source>
        <translation>Falsche Rohrdimension gefunden.</translation>
    </message>
    <message>
        <location filename="../modules/t_intersection_analyzer.py" line="282"/>
        <source>Connecting pipe has a larger diameter than the main pipe.</source>
        <translation>Anschlussleitung hat größere Dimension als Hauptleitung.</translation>
    </message>
</context>
<context>
    <name>progress_bar</name>
    <message>
        <location filename="../massenermittlung.py" line="251"/>
        <source>Performing bulk assessment...</source>
        <translation>Verarbeite Leitungsnetz...</translation>
    </message>
    <message>
        <location filename="../modules/general.py" line="497"/>
        <source>Writing results to new layer...</source>
        <translation>Schreibe Ergebnisse in neuen Layer...</translation>
    </message>
    <message>
        <location filename="../modules/poi_classifier.py" line="62"/>
        <source>Collecting points...</source>
        <translation>Zu untersuchende Punkte werden gesammelt...</translation>
    </message>
    <message>
        <location filename="../modules/poi_classifier.py" line="86"/>
        <source>Analyzing points...</source>
        <translation>Untersuche Punkte...</translation>
    </message>
</context>
<context>
    <name>summary</name>
    <message>
        <location filename="../modules/logs_and_errors.py" line="161"/>
        <source>Type field not found in new layer.</source>
        <translation>Konnte die Spalte für den Formteiltyp nicht finden.</translation>
    </message>
    <message>
        <location filename="../modules/logs_and_errors.py" line="174"/>
        <source>Failed to get type counts from new layer.</source>
        <translation>Konnte die Anlagenteile nicht auszählen.</translation>
    </message>
    <message>
        <location filename="../modules/logs_and_errors.py" line="155"/>
        <source>Layer &apos;{0}&apos; analyzed:</source>
        <translation>Layer '{0}' untersucht:</translation>
    </message>
</context>
</TS>
